intro-to-git/activity-git-history.md

```diff
@@ -0,0 +1,93 @@
+# Activity: git History
+
+<!-- Can be async -->
+<!-- Written to be synchronous -->
+<!-- Ideal Format: Small groups, large groups, roundtables, etc. -->
+
+## Goal
+
+Our goal is to practice the process of making commits as a group. We will also practice collaborating in Git as a group.
+
+Our goal is to practice this without actually typing and running these commands in the terminal; however, we should practice recognizing what commands we _would_ type.
```

Comment:
Is there a reason we can't create a practice repo for them to work on? Practicing on real-world examples is helpful for remembering important details.

Reply:
👍 

For homework, we'll be assigning a whole thing with a practice repo or using an interactive site made for this. This is mostly an activity; kinda wanna see them get the process down before muscle memory.